CHANGELOG.rst

```diff
@@ -6,7 +6,7 @@ community.aws Release Notes
 
 
 v3.1.1
-======
+=======
```

Comment:
I guess there is something wrong due to solving the merge conflict.

You need to keep the info info related to 3.1.1:

```
  3.1.1:
    changes:
      minor_changes:
      - bump the release version of the amazon.aws collection from 3.1.0 to 3.1.1
        because of a bug that occurred while uploading to Galaxy.
    fragments:
    - bump_release_version.yml
    release_date: '2022-02-15'
```

And add only what 2.2.0 introduces.

Reply:
There is also missing the changelog.yml file update.